website/content/docs/index.mdx

```diff
@@ -9,11 +9,3 @@ page_title: Documentation
 [Packer](https://www.packer.io/) is an open source tool that enables you to create identical machine images for multiple platforms from a single source template. A common use case is creating "golden images" that teams across an organization can use in cloud infrastructure.
 
 To install Packer and learn the standard Packer workflow, try the [Get Started tutorials](https://learn.hashicorp.com/packer).
-
-## HCP Packer
-
```

Comment:
you know, I. see why you pulled out ALL of this, but I think a link to the HCP Packer docs on this page isn't necessarily a bad thing. Maybe we could do some shortened version. 

"The HCP Packer registry stores metadata about your images. You can use the registry to track information about golden images from your Packer builds, clearly designate which images are appropriate for test and production environments, and query for the right images to use in both Packer and Terraform configurations.

To get started, visit the [HCP Packer documentation](https://cloud.hashicorp.com/docs/packer) or try the [Get Started with HCP Packer tutorials](https://learn.hashicorp.com/collections/packer/hcp-get-started)."

Reply:
Okay I like this shorten version. I'll add that in.